index.html

```diff
@@ -37,6 +37,109 @@ <h2> Hey There. I’m Alex</h2>
                 </div>
             </div>
         </section>
+        <!-- second part -->
+        <section class="works">
+            <h1 class="recent">My Recent Works</h1>
+            <div class="gridcontainer1">
+                <!-- grid for cardcontainer -->
+                <div class="card1">
+                    <img src="firstimage.png">
+                    <h2 class="stories"> Multi-Post Stories</h2>
+                    <p class="selection">A daily selection of privately personalized reads; no accounts or sign-ups
+                        required. has been
+                        the industry's standard dummy text ever since the 1500s, when an unknown printer took a
+                        standard dummy text.</p>
+                </div>
+                <li class="listbutton">
+                    <ul><button class="button1"> css</button></ul>
+                    <ul><button class="button2"> html</button></ul>
+                    <ul><button class="button3"> bootstrap</button></ul>
+                    <ul><button class="button4"> Ruby</button></ul>
+                </li>
+                <button class="button5"> See Project</button>
+                <section class="listproject">
+                    <p class="art">Profesional Art Printing Data</p>
```

Comment:
This is a header or title. Just as you used an `h2` tag on line 47, kindly consider using same `h2` tag or any other tag aside an `h1` instead of the `p` tag. 
Several of this instances can be found on lines 73, 86, 99, 111, 123 etc.